libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -57,20 +56,22 @@ const (
 
 // New constructs a new add_host_metadata processor.
 func New(cfg *config.C) (processors.Processor, error) {
-	config := defaultConfig()
-	if err := cfg.Unpack(&config); err != nil {
-		return nil, errors.Wrapf(err, "fail to unpack the %v configuration", processorName)
+	c := defaultConfig()
+	if err := cfg.Unpack(&c); err != nil {
+		return nil, fmt.Errorf("fail to unpack the %v configuration: %w", processorName, err)
```

Comment:
The change on this line is unrelated to this PR.  It's been made to please the linter.